entrypoint.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+set -e
+
```

Comment:
I can see that the action metadata is not yet there. what type of action is going to be? Docker type? If so, wouldn't make sense that this entrypoint.sh which in fact looks like a setup script is done using a Dockerfile and hence prepared at build-time instead of at runtime?